src/main/java/multiversx/Transaction.java

```diff
@@ -22,6 +22,8 @@
 
 public class Transaction {
     public static final int VERSION = 1;
```

Comment:
I think we can have the default version = 2

Reply:
I would leave it as is is, since it's clearer